core/src/main/java/com/sequenceiq/cloudbreak/service/cluster/flow/EmailSenderService.java

```diff
@@ -50,33 +50,111 @@
     private UserDetailsService userDetailsService;
 
     @Async
-    public void sendSuccessEmail(String email, String ambariServer) {
+    public void sendProvisioningSuccessEmail(String email, String ambariServer) {
         CbUser user = userDetailsService.getDetails(email, UserFilterField.USERID);
-        sendEmail(user, successClusterInstallerMailTemplatePath, getEmailModel(user.getGivenName(), "SUCCESS", ambariServer));
+        String text = String.format("Your cluster is ready to use. You can log into the Ambari UI %s:8080 using the configured username/password.",
+                ambariServer);
+        sendEmail(user, successClusterMailTemplatePath, "cloudbreak cluster installation", getEmailModel(user.getGivenName(), "SUCCESS", ambariServer, text,
+                "Cloudbreak Cluster Install Success"));
     }
 
     @Async
-    public void sendFailureEmail(String email) {
+    public void sendProvisioningFailureEmail(String email) {
         CbUser user = userDetailsService.getDetails(email, UserFilterField.USERID);
-        sendEmail(user, failedClusterInstallerMailTemplatePath, getEmailModel(user.getGivenName(), "FAILED", null));
+        String text = "Something went terribly wrong "
+                + "- we are happy to help, please let us know your cluster details, time, etc - and we will check the logs and get a fix for you.";
+        sendEmail(user, failedClusterMailTemplatePath, "cloudbreak cluster installation", getEmailModel(user.getGivenName(), "FAILED", null, text,
+                "Cloudbreak Cluster Install Failed"));
     }
 
-    private void sendEmail(CbUser user, String template, Map<String, Object> model) {
+    @Async
+    public void sendStartSuccessEmail(String email, String ambariServer) {
+        CbUser user = userDetailsService.getDetails(email, UserFilterField.USERID);
+        String text = String.format("Your cluster is ready to use after the start. " +
+                        "You can log into the Ambari UI %s:8080 using the configured username/password.", ambariServer);
+        sendEmail(user, successClusterMailTemplatePath, "cloudbreak cluster start", getEmailModel(user.getGivenName(), "SUCCESS", ambariServer, text,
+                "Cloudbreak Cluster Start Success"));
+    }
+
+    @Async
+    public void sendStartFailureEmail(String email, String ambariServer) {
+        CbUser user = userDetailsService.getDetails(email, UserFilterField.USERID);
+        String text = String.format("Your cluster start failed. "
+                        + "- we are happy to help, please let us know your cluster details, time, etc - and we will check the logs and get a fix for you.");
+        sendEmail(user, failedClusterMailTemplatePath, "cloudbreak cluster start", getEmailModel(user.getGivenName(), "FAILED", ambariServer, text,
+                "Cloudbreak Cluster Start Failed"));
+    }
+
+    @Async
+    public void sendStopSuccessEmail(String email, String ambariServer) {
```

Comment:
Two functions could be enough with different parameterized message and title.
One for sending success template and the other one for sending failure template with different parameters and messages.